### DISCUSSION TITLE
Typos

### DISCUSSION THREAD

**Discussion by amake** [fabrizioschiavi/pragmatapro-semiotics#5] [General]
1. How should typos be handled? I have commit access (thanks!) but I imagine there is master data outside of this repo that needs updating, so I assume I should not fix them in the data here myself.
2. What counts as a typo? I assume the description data is meant to be English, and based on the word "color" that it is US English. 

The item I noticed that prompted these questions:

<img width="826" alt="image" src="https://github.com/fabrizioschiavi/pragmatapro-semiotics/assets/2172537/267e0fbc-c9dc-4949-9e83-ffcce92ba5bf">

**Comment by fabrizioschiavi** [comment]
Thanks @amake for what you done for this repo!

The typos can handle opening the [ods](https://github.com/fabrizioschiavi/pragmatapro-semiotics/blob/main/Symbols.ods) file with LibreOffice, then save or export it in [csv](https://github.com/fabrizioschiavi/pragmatapro-semiotics/blob/main/Symbols.csv) file too. My master data is only the ods file.

I think the only way we should create a highly effective document focused on pinpointing the most relevant keywords. This file will serve as a valuable resource for identifying symbols with precision and maximizing their utility.
At the moment I would like to start in US English but in the future I can't exclude to includes other languages in this small database.
Do you agree?

**Reply by amake** [reply]
Sure, sounds good.

(I thought maybe the ODS and CSV were generated from the font files, but I see that's not the case.)

**Reply by fabrizioschiavi** [reply]
Unfortunately no 🙁
Artisanally generated only
